Same error as #97 in non-English environment

The same error as #97 occurs in non-English environment.

https://github.com/jline/jline3/commit/12219fa5a5fd56dbe0731519032c862fc4e39388 works well in `LANG=C`, but in `LANG=ja_JP.UTF-8` it throws

```
Exception in thread "main" java.io.IOError: java.io.IOException: Error executing 'stty icrnl ixon echo icanon iexten min 1 time 0': stty: 標準入力: 要求された処理の全てを実行することができません
	at org.jline.terminal.impl.AbstractPosixTerminal.setAttributes(AbstractPosixTerminal.java:54)
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:628)
	at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:404)
	at com.github.imas.rdflint.InteractiveMode.execute(InteractiveMode.java:83)
	at com.github.imas.rdflint.RdfLint.main(RdfLint.java:103)
Caused by: java.io.IOException: Error executing 'stty icrnl ixon echo icanon iexten min 1 time 0': stty: 標準入力: 要求された処理の全てを実行することができません
	at org.jline.utils.ExecHelper.exec(ExecHelper.java:42)
	at org.jline.terminal.impl.ExecPty.doSetAttr(ExecPty.java:101)
	at org.jline.terminal.impl.AbstractPty.setAttr(AbstractPty.java:21)
	at org.jline.terminal.impl.AbstractPosixTerminal.setAttributes(AbstractPosixTerminal.java:52)
	... 4 more
```

In the error message `要求された処理の全てを実行することができません` means `unable to perform all requested operations` in Japanese.

--- a/terminal/src/main/java/org/jline/terminal/impl/ExecPty.java
+++ b/terminal/src/main/java/org/jline/terminal/impl/ExecPty.java
@@ -97,19 +97,7 @@ public class ExecPty extends AbstractPty implements Pty {
                 commands.add(1, OSUtils.STTY_F_OPTION);
                 commands.add(2, getName());
             }
-            try {
-                exec(system, commands.toArray(new String[commands.size()]));
-            } catch (IOException e) {
-                // Handle partial failures with GNU stty, see #97
-                if (e.toString().contains("unable to perform all requested operations")) {
-                    commands = getFlagsToSet(attr, getAttr());
-                    if (!commands.isEmpty()) {
-                        throw new IOException("Could not set the following flags: " + String.join(", ", commands), e);
-                    }
-                } else {
-                    throw e;
-                }
-            }
+            exec(system, commands.toArray(new String[commands.size()]));
         }
     }
 

--- a/terminal/src/main/java/org/jline/utils/OSUtils.java
+++ b/terminal/src/main/java/org/jline/utils/OSUtils.java
@@ -66,12 +66,11 @@ public class OSUtils {
         } else {
             tty = "tty";
             stty = "stty";
+            sttyfopt = "-F";
             infocmp = "infocmp";
             if (IS_OSX) {
+                stty = "/bin/stty";
                 sttyfopt = "-f";
-            }
-            else {
-                sttyfopt = "-F";
             }
         }
         TTY_COMMAND = tty;

